verifiable/did_doc.go

```diff
@@ -0,0 +1,20 @@
+package verifiable
+
+// DIDDocument defines current supported did doc model.
+type DIDDocument struct {
+	Context []string  `json:"context"`
+	ID      string    `json:"id"`
+	Service []Service `json:"service"`
+}
+
+// Service DID document service field.
+type Service struct {
+	ID              string `json:"id"`
+	Type            string `json:"type"`
+	ServiceEndpoint string `json:"serviceEndpoint"`
```

Comment:
In most places, we use `snake_case`. https://github.com/iden3/go-schema-processor/blob/main/verifiable/proof.go But here we use `camelCase`. Can we define a common standard for JSON tags? 
P.S. Here we use `snake_case` and `camelCase` together.